xbmc/dialogs/GUIDialogKeyboardGeneric.cpp

```diff
@@ -257,6 +313,17 @@ bool CGUIDialogKeyboardGeneric::OnMessage(CGUIMessage& message)
         OnOK();
       return true;
     }
+  case GUI_CODINGTABLE_LOOKUP_COMPLETED:
+    {
+      int response = message.GetParam1();
+      auto words = m_codingtable->GetResponse(response);
```

Comment:
This is only used if the condition below is true so this can be moved inside the conditional block.